sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -6191,7 +6350,7 @@
             "encrypted": false,
             "redrawOn": "",
             "customDefaultValue": "",
-            "calculateValue": "if (data.hasJointDesignatedInstitution === \"no\" || data.eslEligibility === \"20OrMore\" || \r\n(data.deliveredOnlineAlsoOnsite === \"no\" && data.sameOnlineCreditsEarned === \"no\" && earnAcademicCreditsOtherInstitution === \"no\") \r\n|| (data.courseLoadCalculation === \"hours\" && data.minHoursWeek === \"no\" && data.isAviationProgram === \"no\") || (data.courseLoadCalculation === \"hours\" && data.minHoursWeek === \"no\" && data.isAviationProgram === \"yes\" && data.minHoursWeekAvi === \"no\") \r\n|| (data.hasWILComponent === \"yes\" && data.isWILApproved === \"no\") || (data.hasWILComponent === \"yes\" && data.isWILApproved === \"yes\" && data.wilProgramEligibility === \"no\") \r\n|| (data.hasTravel === \"yes\" && data.travelProgramEligibility === \"no\") || (data.hasIntlExchange === \"yes\" && data.intlExchangeProgramEligibility === \"no\")) {\r\n  value = \"Pending\";\r\n}\r\nelse {\r\n  value = \"Approved\";\r\n}",
+            "calculateValue": "if (data.hasJointDesignatedInstitution === \"no\" || \r\n\tdata.eslEligibility === \"20OrMore\" || \r\n\tdata.entranceRequirements.noneOfTheAboveEntranceRequirements === true ||\r\n\t(data.deliveredOnlineAlsoOnsite === \"no\" && data.sameOnlineCreditsEarned === \"no\" && data.earnAcademicCreditsOtherInstitution === \"no\") || \r\n\t(data.courseLoadCalculation === \"hours\" && data.minHoursWeek === \"no\" && data.isAviationProgram === \"no\") ||\r\n\t(data.courseLoadCalculation === \"hours\" && data.minHoursWeek === \"no\" && data.isAviationProgram === \"yes\" && data.minHoursWeekAvi === \"no\") || \r\n\t(data.hasWILComponent === \"yes\" && data.isWILApproved === \"no\") ||\r\n\t(data.hasWILComponent === \"yes\" && data.isWILApproved === \"yes\" && data.wilProgramEligibility === \"no\") ||\r\n\t(data.hasTravel === \"yes\" && data.travelProgramEligibility === \"no\") ||\r\n\t(data.hasIntlExchange === \"yes\" && data.intlExchangeProgramEligibility === \"no\")) {\r\n  value = \"Pending\";\r\n}\r\nelse {\r\n  value = \"Approved\";\r\n}\r\n\r\nconsole.log(\"Value \"+value)",
```

Comment:
For form.io javascript code, I would always prefer leaving the format to VS Code, as below.
Also, I believe that we target ES6 for form.io hence we can use `const` instead of var.

```ts
const shouldSelectOneOfTheAboveOrAnotherOption =
  (data.entranceRequirements.minHighSchool ||
    data.entranceRequirements.hasMinimumAge ||
    data.entranceRequirements.requirementsByInstitution ||
    data.entranceRequirements.requirementsByBCITA) &&
  data.entranceRequirements.noneOfTheAboveEntranceRequirements;

const shouldSelectAtLeastOneOption =
  !data.entranceRequirements.minHighSchool &&
  !data.entranceRequirements.hasMinimumAge &&
  !data.entranceRequirements.requirementsByInstitution &&
  !data.entranceRequirements.requirementsByBCITA &&
  !data.entranceRequirements.noneOfTheAboveEntranceRequirements;

valid = shouldSelectAtLeastOneOption
  ? "An entrance requirement is required."
  : shouldSelectOneOfTheAboveOrAnotherOption
  ? "Please select one of the entrance requirements or 'None of the above'."
  : true;
```